QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -10,7 +10,7 @@
 
 public class AutoLoader extends InstinctModule {
     private final BooleanStatus running;
-    public static final BooleanInput crateInPosition = BooleanMixing.createDispatch(Igneous.makeDigitalInput(3), Igneous.globalPeriodic);
+    public static final BooleanInput crateInPosition = BooleanMixing.createDispatch(Igneous.makeDigitalInput(5), Igneous.globalPeriodic);
```

Comment:
As of last time I was in, Anna told me that they'd put the limit switches in 2 and 4, and left the crate port as 3.

Assuming that this is spurred by getting Helios to work, why did they change it?